pulp/pulp.py

```diff
@@ -144,28 +144,28 @@
 try:  # allow Python 2/3 compatibility
     maketrans = str.maketrans
 except AttributeError:
-    from string import maketrans
+    from string import maketrans  # type: ignore[attr-defined,no-redef]
 
 _DICT_TYPE = dict
 
 if sys.platform not in ["cli"]:
```

Comment:
There is no `sys.platform` which is `cli` any more. I presume this was IronPython. Why not simplify and just remove `_DICT_TYPE` as per https://github.com/coin-or/pulp/pull/807/files#diff-b2a258edfe0fa294f032a978656a4082842558faa98b0950d9f68c50f546bbe2L151